### DISCUSSION TITLE
Is there a setting in laragon that can enable only one instance of Laragon to be running ?

### DISCUSSION THREAD

**Discussion by kressly** [leokhoa/laragon#788] [Q&A]
I notice that when i give laragon to someone, they lauchn it sometimes twice and get confused. Is there a setting in laragon that can enable only one instance of Laragon to be running so that once one instance is running, even if they mistakenly click on it again it wont run another one unless the old one is exited  ?

**Comment by Pen-y-Fan** [comment]
Laragon is designed to be portable, multiple instances can be started. The system tray icon can also get lost in the other icons. I have written a blog to pin the [Laragon system tray icon](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-3/). I hope this helps.

**Reply by kressly** [reply]
Thank you for ur fast response
